x-pack/docs/en/rest-api/security/oidc-prepare-authentication-api.asciidoc

```diff
@@ -57,20 +57,19 @@ POST /_security/oidc/prepare
 }
 --------------------------------------------------
 // CONSOLE
-// TEST[skip:These are properly tested in the OpenIDConnectIT suite]
-
 The following example output of the response contains the URI pointing to the Authorization Endpoint of the
 OpenID Connect Provider with all the parameters of the Authentication Request, as HTTP GET parameters
 
 [source,js]
 --------------------------------------------------
 {
-  "redirect" : "https://op-provider.org/login?scope=openid&response_type=code&redirect_uri=http%3A%2F%2Foidc-kibana.elastic.co%3A5603%2Fkmi%2Fapi%2Fsecurity%2Fv1%2Foidc&state=4dbrihtIAt3wBTwo6DxK-vdk-sSyDBV8Yf0AjdkdT5I&nonce=WaBPH0KqPVdG5HHdSxPRjfoZbXMCicm5v1OiAj0DUFM&client_id=0o43gasov3TxMWJOt839",
+  "redirect" : "http://127.0.0.1:8080/c2id-login?scope=openid&response_type=id_token&redirect_uri=https%3A%2F%2Fmy.fantastic.rp%2Fcb&state=4dbrihtIAt3wBTwo6DxK-vdk-sSyDBV8Yf0AjdkdT5I&nonce=WaBPH0KqPVdG5HHdSxPRjfoZbXMCicm5v1OiAj0DUFM&client_id=elasticsearch-rp",
```

Comment:
This still fails as the substitution doesn't seem to work for the value of the redirect parameter :/ I'll keep looking

Reply:
```
java.lang.AssertionError: Failure at [en/rest-api/security/oidc-prepare-authentication-api:72]: $body didn't match expected value:
                         $body: 
                           nonce: same [jbmQm8FRAP-6a3SE_vjjnoReu6nyOcUNjXzF5EcX1Ao]
                        redirect: expected [http://127.0.0.1:8080/c2id-login?login_hint=this_is_an_opaque_string&scope=openid&response_type=id_token&redirect_uri=https%3A%2F%2Fmy.fantastic.rp%2Fcb&state=$body.state&nonce=$body.nonce&client_id=elasticsearch-rp] but was [http://127.0.0.1:8080/c2id-login?login_hint=this_is_an_opaque_string&scope=openid&response_type=id_token&redirect_uri=https%3A%2F%2Fmy.fantastic.rp%2Fcb&state=D6aKfaFTPe4b9ymmoDBDGu25dUcxQzUfQeYmVewhLa8&nonce=jbmQm8FRAP-6a3SE_vjjnoReu6nyOcUNjXzF5EcX1Ao&client_id=elasticsearch-rp]
                           state: same [D6aKfaFTPe4b9ymmoDBDGu25dUcxQzUfQeYmVewhLa8]
```


redirect: expected 
```
[http://127.0.0.1:8080/c2id-login?login_hint=this_is_an_opaque_string&scope=openid&response_type=id_token&redirect_uri=https%3A%2F%2Fmy.fantastic.rp%2Fcb&state=$body.state&nonce=$body.nonce&client_id=elasticsearch-rp] 
```
but was 
```
[http://127.0.0.1:8080/c2id-login?login_hint=this_is_an_opaque_string&scope=openid&response_type=id_token&redirect_uri=https%3A%2F%2Fmy.fantastic.rp%2Fcb&state=D6aKfaFTPe4b9ymmoDBDGu25dUcxQzUfQeYmVewhLa8&nonce=jbmQm8FRAP-6a3SE_vjjnoReu6nyOcUNjXzF5EcX1Ao&client_id=elasticsearch-rp]
```